src/Http/Response/BadRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
```

Comment:
Don't think this file should be here. Maybe need to rebase?

Reply:
Yeah, I had pulled some of some of Xavier's json response classes from his pr that got merged into minor (#4275). I don't need this one for the module, so I'll drop it, but I think i do need the `Forbidden` and `Invalid` classes.

Reply:
Ah okay, didn't realize that they had gone to minor and not major.